CompetitionTemplate/src/competition/subsystems/arm/arm_commands/LowerArm.java

```diff
@@ -0,0 +1,31 @@
+package competition.subsystems.arm.arm_commands;
+import competition.subsystems.arm.ArmSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class LowerArm extends BaseCommand {
+    ArmSubsystem armSubsystem;
+
+    public LowerArm(ArmSubsystem armSubsystem) {
+        this.armSubsystem = armSubsystem;
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        if (armSubsystem.isArmAtMaximumHeight()){
```

Comment:
Also in general you might get more millage out of this check being in the subsystem so that nobody can ask to lower the arm when it's already as low as it can go.